### ISSUE TITLE
Sync: Trakt.tv doesn't sync TV Shows on Chrome OS.

### ISSUE THREAD

**Issue by cecoates** [UweTrottmann/SeriesGuide#584]
Steps to reproduce.

1. Enable the Play Store on your Chrome OS device.
2. Make an Android browser default, such as Chrome Beta (credentials for apps like SeriesGuide will only work when using a browser installed from the Play Store/the integrated browser)
3. Open SeriesGuide, and login via Trakt.tv
4. Wait for sync to finish.

Movies will load as expected, both the watchlist and collection. However, TV Shows will remain empty.

No errors display, and syncing seems successful, but the TV Shows just never get added.

Syncing via SeriesGuide Cloud will load both movies and TV Shows as expected.

Interestingly, once TV Shows ARE synced either via SeriesGuide Cloud (as long as you disconnect it after syncing so it doesn't conflict with Trakt.tv) or via an exported backup file, it WILL sync updates such as marking an episode watched, or if a new show is added.

In other words, even though SeriesGuide won't initially load the TV Shows from Trakt, it seems to sync episodes you mark watched after that perfectly fine.

Asus C302 Version 69.0.3497.120 (Official Build) (64-bit)
SeriesGuide version 45-beta2

Completely anecdotally, there seems to be some sort of issue where the status text stops on TVDB for longer than expected. But not an actual error, as it eventually moves on. Is that where the data gets pulled? Maybe it gets :-( faced when too much info is being requested at once?

**Comment by UweTrottmann** [comment]
TV shows are not added automatically from your trakt profile as SeriesGuide handles them differently than trakt. Find your watched, collected and shows on your watchlist from the discover screen (tap the + button from the show list after connecting trakt).

If SeriesGuide Cloud is connected, trakt two-way sync is disabled.
